html/partials/scope_header.html.haml

```diff
@@ -0,0 +1,19 @@
+- meta = @description[scope].meta
+.row
+  .col-xs-1
+    %img.over.scope_logo_big{ |
+      "data-content" => "<p>#{scope_help(scope)}</p>", |
+      "data-toggle" => "popover", |
+      "src" => "assets/logo-#{scope.tr("_", "-")}.png", |
+      "title" => "", |
+      "data-original-title" => title, |
+      "style" => "top: 125px;" |
+    } |
+    %span.toggle{"title" => "Collapse/Expand"}
```

Comment:
Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace